var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -3,10 +3,12 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack.build_systems.autotools import AutotoolsBuilder
+from spack.build_systems.cmake import CMakeBuilder
 from spack.package import *
 
 
-class Freetype(AutotoolsPackage):
+class Freetype(CMakePackage):
```

Comment:
Shouldn't this include `AutotoolsPackage`? Even when adding it, `autotools` does not show up as a valid choice for `build_system`, though:
```
==> Error: invalid values for variant "build_system" in package "freetype": ['autotools']
```

Reply:
CMake is the buildsystem for this package now. We no longer want to inherit autotools at that point.

Reply:
Does it hurt to keep Autotools? It's still Freetype's default build system.

Reply:
I don't personally see utility in maintaining two build systems when one should suffice and provides improved cross platform support. While Spack does support a package having multiple builders, I think avoiding it where possible is ideal.